services/src/test/scala/cromwell/pipeline/service/ProjectConfigurationServiceTest.scala

```diff
@@ -159,5 +171,51 @@ class ProjectConfigurationServiceTest extends AsyncWordSpec with Matchers with M
           .map(_ should have.message("Access denied. You  not owner of the project"))
       }
     }
+    "build configuration" should {
+
+      "return success message for request" taggedAs Service in {
+
+        when(configurationRepository.getAllByProjectId(projectId)).thenReturn(Future.successful(Seq()))
+        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future.successful(dummyProject))
+        when(projectVersioning.getFile(dummyProject, projectFile.path, optionVersion))
+          .thenReturn(Future.successful(Right(projectFile)))
+        when(womTool.inputsToList(projectFileContent.content)).thenReturn(Right(Nil))
+
+        configurationService
+          .buildConfiguration(projectId, projectFile.path, optionVersion, userId)
+          .map(
+            builtConfiguration =>
+              builtConfiguration shouldBe ProjectConfiguration(
+                id = builtConfiguration.id,
+                projectId = projectId,
+                active = true,
+                projectFileConfigurations = List(ProjectFileConfiguration(projectFile.path, Nil)),
+                version = ProjectConfigurationVersion.defaultVersion
+              )
```

Comment:
I think you should exclude configuration from test and use created object.

Reply:
Can you be more specific? If you mean to use dummyConf, how to compare it by random Id?

Reply:
Ok, in this case used id from builtConguration, maybe I was wrong and you shouldn`t move this pice of code. Sorry.